2214/2/battle_expert.rb

```diff
@@ -0,0 +1,48 @@
+require 'terminal-table'
+require_relative 'popular_words_counter'
+require_relative 'battler'
+require_relative 'battle'
+
+class BattleExpert
+  BATTLES_FOLDER = 'Battles'.freeze
+  def describe_battlers(top_bad_words)
+    puts Terminal::Table.new rows: sorted_battlers[0...top_bad_words].map { |battler| battler_as_row(battler) }
```

Comment:
Since you use `#battler_as_row` only in this method, I wouldn't even extract it to a separate method.

`#battler_as_row` method looks fine but I just don't like private methods that receive arguments.

So feel free to decide what to do here.